series-tvdb.en.yaml

```diff
@@ -659,3 +659,69 @@ entries:
     seasons:
       - season: 1
         anilist-id: 20966
+        
+    - title: "FLCL"
+    synonyms:
+      - "FLCL Alternative"
+      - "FLCL Progressive"
+    seasons:
+      - season: 1
+        anilist-id: 227
+      - season: 2
+        anilist-id: 21748
+      - season: 3
+        anilist-id: 21746
+        
+  - title: "I Tried Asking While Kowtowing"
+    synonyms:
+      - "<DOGEZA>I Tried Asking While Kowtowing."
+    seasons:
+      - season: 1
+        anilist-id: 122137
+        
+  - title: "Sakamichi no Apollon: Kids on the Slope"
+    synonyms:
+      - "Sakamichi no Apollon"
+      - "Kids on the Slope"
+    seasons:
+      - season: 1
+        anilist-id: 12531
+        
+  - title: "SK8 the Infinity"
+    synonyms:
+      - "SK8"
+    seasons:
+      - season: 1
+        anilist-id: 124153
+        
+  - title: "A Silent Voice: The Movie"
+    synonyms:
+      - "A Silent Voice"
+    seasons:
+      - season: 1
+        anilist-id: 20954
+        
+  - title: "Space Pirate Captain Harlock"
```

Comment:
Another movie, rejected
Also forgot the most basic synonym `Uchuu Kaizoku Captain Harlock`